geode-protobuf/src/main/proto/basicTypes.proto

```diff
@@ -52,7 +52,12 @@ message CallbackArguments {
 
 message Region {
```

Comment:
One would want to avoid just dumping fields into the response object without context. Given all fields are related to the Region, it is better suited to use a Region message.